apps/ejabberd/src/mod_admin_extra_vcard.erl

```diff
@@ -113,21 +113,44 @@ commands() ->
 %%%
 
 get_vcard(User, Host, Name) ->
```

Comment:
Remember to add specs for exported functions and also change them when you change function's result value